examples/passport-oauth2-login/src/authentication-strategies/auth-action.provider.ts

```diff
@@ -0,0 +1,84 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/authentication
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Getter, inject, Provider, Setter} from '@loopback/context';
```

Comment:
Hmm, wouldn't this action be provided by the `@loopback/authentication` module?

Reply:
agreed